modules/network-observability-operator-install.adoc

```diff
@@ -25,26 +30,18 @@ This documentation assumes that your `LokiStack` instance name is `loki`. Using
-** Set the *type* to `KAFKA`.
-** Set the *address* as `kafka-cluster-kafka-bootstrap.netobserv`.
-** Set the *topic* as `netobserv-flows-export`. The Operator exports all flows to the configured Kafka topic.
-** Set the following *tls* specifications:
-*** *certFile*: `service-ca.crt`, *name*: `kafka-gateway-ca-bundle`, and *type*: `configmap`.
-+
-You can also configure this option at a later time by directly editing the YAML. For more information, see _Export enriched network flow data_.
-* *loki.enable*: Set to `true`.
-* *loki.url*: Since authentication is specified separately, this URL needs to be updated to `https://loki-gateway-http.netobserv.svc:8080/api/logs/v1/network`. The first part of the URL, "loki", should match the name of your LokiStack.
-* *loki.statusUrl*: Set this to `https://loki-query-frontend-http.netobserv.svc:3100/`. The first part of the URL, "loki", should match the name of your LokiStack.
-* *loki.authToken*: Select the `FORWARD` value.
-* *tls.enable*: Verify that the box is checked so it is enabled.
-* *statusTls*: The `enable` value is false by default.
+. Navigate to the *Flow Collector* tab, and click *Create FlowCollector*. Make the following selections in the form view:
+.. *spec.agent.ebpf.Sampling*: Specify a sampling size for flows. Lower sampling sizes will have higher impact on resource utilization. For more information, see the "FlowCollector API reference", `spec.agent.ebpf`.
+.. If you are using Loki, set the following specifications:
+... *spec.loki.enable*: Check the box to enable storing flows in Loki.
+... *spec.loki.url*: Since authentication is specified separately, this URL needs to be updated to `https://loki-gateway-http.netobserv.svc:8080/api/logs/v1/network`. The first part of the URL, "loki", should match the name of your `LokiStack`.
+... *spec.loki.authToken*: Select the `FORWARD` value.
+... *spec.loki.statusUrl*: Set this to `https://loki-query-frontend-http.netobserv.svc:3100/`. The first part of the URL, "loki", should match the name of your `LokiStack`.
+... *spec.loki.tls.enable*: Verify that the box is checked so it is enabled.
+... *spec.loki.statusTls*: The `enable` value is false by default.
 +
 For the first part of the certificate reference names: `loki-gateway-ca-bundle`, `loki-ca-bundle`, and `loki-query-frontend-http`,`loki`, should match the name of your `LokiStack`.
+.. Optional: You might want to configure other optional settings before the next step of creating the `FlowCollector`. For example, if you choose not to use Loki, then you can configure exporting flows to Kafka or IPFIX by directly editing the YAML. For more information, see "Export enriched network flow data to Kafka and IPFIX" and more in the "Important Flow Collector configuration considerations" following this section.
```

Comment:
`by directly editing the YAML` should be removed. The YAML is not created yet

Reply:
Addressed